operator/controllers/suite_test.go

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
do you think that /operator should have more unit tests in general?
we only have suite_test.go in the whole initial draft.

Reply:
definitely
see [1]